code/datums/components/label.dm

```diff
@@ -72,6 +73,12 @@
 	* user: The mob exmaining the parent.
 	* examine_list: The current list of text getting passed from the parent's normal examine() proc.
 */
+
+///Reapplies label when update_name() is called on the parent object. Attempts to remove it first just in case.
+/datum/component/label/proc/OnUpdateName()
```

Comment:
I know it's kind of out of scope, but could you please convert all of the proc names in this file to be in camel_case?

Reply:
Sure, why not?